javascript/core/recipes-i/round-numbers-to-n-decimals.md

```diff
@@ -1,28 +1,15 @@
 ---
 author: catalin
 
-levels:
-
-  - basic
-
-  - beginner
-
 type: normal
 
 category: how to
 
-standards:
-
-  javascript.write-expressions.0: 10
-
-  javascript.evaluate-expressions.3: 10
-
 aspects:
   - introduction
   - workout
 
 
-
 ---
 
 # Round numbers to `N` decimals 
```

Comment:
should we capitalize headlines? (I've been doing this for insights that I update/fix)

```suggestion
# Round Numbers to `N` Decimals
```

Reply:
I'm fine either way, you decide